src/Compilers/CSharp/Portable/Symbols/Source/SourceMethodSymbolWithAttributes.cs

```diff
@@ -1271,5 +1271,45 @@ internal override System.Reflection.MethodImplAttributes ImplementationAttribute
                 return result;
             }
         }
+
+        private ImmutableArray<BoundAttribute> GetBoundAttributes(OneOrMany<SyntaxList<AttributeListSyntax>> declarations, AttributeLocation symbolPart)
```

Comment:
It looks like we use this to bind the attributes during flow analysis. Should we instead make the bound attributes a property of 'BoundLocalFunctionStatement' and access the property during flow analysis?

It seems like a similar issue will come up when default parameter values use local constants. I'll think a bit about it and decide whether this basic idea seems reasonable (doing binding during flow analysis). I'm curious what you think though.